src/pymatgen/core/periodic_table.py

```diff
@@ -1639,9 +1656,18 @@ def get_el_sp(obj: int | SpeciesLike) -> Element | Species | DummySpecies:
             of properties that can be determined.
     """
     # If obj is already an Element or Species, return as is
-    if isinstance(obj, Element | Species | DummySpecies):
```

Comment:
I don't really understand this change?

Reply:
Also fixed by running on linux, but for record-keeping the mypy error on my mac for this was `Unsupported left operand type for | ("type[Element]")  [operator]`